src/main/resources/events-mapping.json

```diff
@@ -1,114 +1,49 @@
+    "message_type": { "type": "keyword", "ignore_above": 100 },
     "timestamp": {
       "type": "date",
+      "format":"rfc3339_lenient||epoch_millis",
+      "ignore_malformed": true, 
       "doc_values": true
     },
     "event_attributes": {
       "properties": {
-        "user_name": {
-          "type": "keyword"
-        },
-        "user_id": {
-          "type": "keyword"
-        },
-        "email": {
-          "type": "keyword"
-        },
-        "price": {
-          "type": "float"
-        },
-        "ip": {
-          "type": "ip",
-          "ignore_malformed": true
-        },
-        "browser": {
-          "type": "text",
-          "fields": {
-            "keyword": {
-              "type": "keyword"
-            }
-          }
+        "dwell_time": { "type": "integer" },
+        "ip": { "type": "ip", "ignore_malformed": true },
+        "browser": { "type": "text",
+          "fields": { "keyword": { "type": "keyword", "ignore_above": 256 } }
         },
         "position": {
           "properties": {
-            "ordinal": {
-              "type": "integer"
-            },
-            "x": {
-              "type": "integer"
-            },
-            "y": {
-              "type": "integer"
-            },
-            "page_depth": {
-              "type": "integer"
-            },
-            "scroll_depth": {
-              "type": "integer"
-            },
-            "trail": {
-              "type": "text",
-              "fields": {
-                "keyword": {
-                  "type": "keyword"
-                }
+            "ordinal": { "type": "integer" },
+            "x": { "type": "integer" },
+            "y": { "type": "integer" },
+            "page_depth": { "type": "integer" },
+            "scroll_depth": { "type": "integer" },
+            "trail": { "type": "text",
+              "fields": { "keyword": { "type": "keyword", "ignore_above": 256 }
               }
             }
           }
         },
         "object": {
           "properties": {
-            "key_value": {
-              "type": "keyword"
-            },
-            "object_id": {
-              "type": "keyword"
-            },
-            "object_type": {
-              "type": "keyword"
-            },
-            "transaction_id": {
-              "type": "keyword"
-            },
-            "name": {
-              "type": "keyword"
-            },
-            "description": {
-              "type": "text",
-              "fields": {
-                "keyword": {
-                  "type": "keyword"
-                }
-              }
-            },
-            "to_user_id": {
-              "type": "keyword"
-            },
-            "object_detail": {
-              "type": "text"
-            }
+            "internal_id": { "type": "keyword" },
+            "object_id": { "type": "keyword", "ignore_above": 256 },
+            "object_type": { "type": "keyword", "ignore_above": 100 },
```

Comment:
are we still doing object_type?  we should chat since we don't have that in the spec, are we missing it?   I guess I was assuming an object_id would tell you everything you needed to know....

Reply:
I think we need it.  At the very least, in the js you can stuff anything into an object.  So, for example, I cram the whole javascript timer in there because it has other cool info that someone might want.
That aside, I'm sure someone will have multiple kinds of things/objects/cats/dogs with the same id.